docs/en/ingest-management/processors/processor-decode_cef.asciidoc

```diff
@@ -0,0 +1,67 @@
+[[decode_cef-processor]]
```

Comment:
I wonder if we want to mention somewhere that this is only available for use with log inputs. Some processors are only available in particular Beats so if you tried to use this on something that's not running in Filebeat then it will fail to run.

It's probably rare that anyone will be affected by this so maybe we want to hide this detail for now. If all processors move into the shipper at some point then this potential footgun goes away.

Reply:
I think we should mention restrictions like "only works with log inputs" but probably avoid mentioning specific Beats or inputs.